web/src/components/FilingsCalendar.tsx

```diff
@@ -125,7 +126,13 @@ export const FilingsCalendar = (props: Props): ReactElement => {
             return (
               <tr key={rowIndex}>
                 {monthIndicesForRow.map((month) => {
-                  return (
+                  const date = getJanOfCurrentYear().add(month, "months");
+
+                  return date.month() < getCurrentDate().month() ? (
+                    <td key={month} className="td-gray-border bg-base-extra-light">
+                      {getMonth(month)}
+                    </td>
```

Comment:
I think that instead of rewriting the entire `td` in this ternary just to add one class name, it would be cleaner to refactor this to use a dynamic className. Then you don't need to duplicate the `td` code and you don't need a ternary 

I went searching for an example, and there's a lot of places. For example, line 67 of `PaymentTypeTable`: 
```
<div className={`${hasError ? "error" : ""} input-error-bar`}>
```

Reply:
I have now created a function, isCalendarMonthLessThanCurrentMonth, and set the className to  that function with a ternary to conditionally render the gray background.